libs/tokens/src/pure/TokenLogo/index.tsx

```diff
@@ -184,12 +195,14 @@ export function TokenLogo({ logoURI, token, className, size = 36, sizeMobile, no
 
   if (noWrap) return tokenContent
 
+  const chainName = token?.chainId ? getChainInfo(token.chainId as SupportedChainId).label : ''
```

Comment:
When `token.chainId` is not from `SupportedChainId` the result of `getChainInfo` will be undefined and app will crash because you try to get `label` from undefined

Reply:
Addressed now.